### DISCUSSION TITLE
InputNumber can be cheated in order to enter symbols or letters

### DISCUSSION THREAD

**Discussion by AlvaroTKT** [primefaces/community#1597] [PrimeReact] [answered]
Hello,

We've encountered a bug while implementing the InputNumber component from Prime React. It's possible to input non-numeric characters when typing double-tap characters (such as an accent "`").

**Steps to reproduce** : This issue can be reproduced using the library's component guide itself. If you focus on the component, type an accent, and then immediately type a letter, the component inserts that letter. At the event level, nothing is triggered (neither event nor e.value).

![image](https://github.com/primefaces/community/assets/124895195/81a89614-5f3a-4cae-bd80-f7c357927778)

Is there any workaround, or is it indeed confirmed as a bug?

**Comment by melloware** [comment]
can you please open a GitHub issue ticket with this bug. I assume PrimeVue has the same issue as PrimeReact?  Try: https://primevue.org/inputnumber/

**Reply by AlvaroTKT** [reply, marked as answer]
Yes, you were right. It can be reprocued in PrimeVue too. I opened an issue by the option here. https://github.com/primefaces/community/issues/1621

**Reply by melloware** [reply]
I moved your ticket to the PrimeReact repo.

**Reply by AlvaroTKT** [reply]
Thank you and sorry. It's my first time creating issues on GitHub.

**Reply by melloware** [reply]
No problem!

**Comment by melloware** [comment]
Fixed in next release hopfully!

**Reply by AlvaroTKT** [reply]
Thanks for the hard work ^^
